### ISSUE TITLE
harvestcraft gardens don't spread

### ISSUE THREAD

**Issue by Lescius** [AllTheMods/ATM-3#169]
From the harvestcraft.cfg file:

```
gardens {
    B:enablegardenSpread=true
    I:gardenSpreadRate=1
    I:gardenspreadMax=1
 }
```

So garden spread is enabled in general, at a low probability, but only if there are no (fewer than one) nearby gardens. However, every garden being ticked will find at least one nearby garden: itself.

See also [BlockBaseGarden:updateTick](https://github.com/MatrexsVigil/harvestcraft/blob/3753ae8bb538b35653a7d631cc7cf50d58102fa2/src/main/java/com/pam/harvestcraft/blocks/blocks/BlockBaseGarden.java)

Tested with ATM3 3.3 by placing several gardens at one per chunk in a flat world, turning random tick rate to 100, and waiting several hours. No spread observed. Then I increased gardenSpreadMax to 2, ran the same test, and saw spread.

**Comment by OrdinatorStouff** [comment]
It was originally intended to have the gardens not grown and possibly not even spawn at all and instead drop pam's seeds from grass but that portion of the config is incomplete. We'll review and decide one way or another (spreading gardens vs grass seeds drops). Thanks for the report.

**Comment by OrdinatorStouff** [comment]
[Fixed in pack version 4.0](https://minecraft.curseforge.com/projects/all-the-mods-3/files/2485186)
